src/coreclr/src/tools/ReadyToRun.SuperIlc/ProcessRunner.cs

```diff
@@ -310,7 +310,10 @@ public bool IsAvailable(ref int progressIndex, ref int failureCount)
         }
         _processInfo.ExitCode = (_processInfo.TimedOut ? TimeoutExitCode : _process.ExitCode);
         _processInfo.Succeeded = (!_processInfo.TimedOut && _processInfo.ExitCode == _processInfo.Parameters.ExpectedExitCode);
-        _logWriter.WriteLine(">>>>");
+        lock (_logWriter)
```

Comment:
Nit - considering how widespread this problem appears to be, would it make sense to create a helper method for the locked log output?

Reply:
Good point, I'll do it.